share/normalize/links.py

```diff
@@ -840,6 +848,30 @@ def _parse(self, obj):
         }
 
 
+class ARKLink(AbstractIRILink):
```

Comment:
Would we want to express arks as URL like we do with DOIs?
I guess the better question is, do we get enough information to?